### ISSUE TITLE
is letting go of a key adressable?

### ISSUE THREAD

**Issue by h3oCharles** [AlexFolland/turbo#1]
hi, so one day i wanted to input "Space" when the spacebar is both in up and down states... is it possible with this?

**Comment by BujuArena** [comment]
I don't know what you mean.  Can you rephrase the question?  You can phrase it verbosely if it helps me understand.

**Comment by h3oCharles** [comment]
holding rope -

tap space (usually called "down" state or sth) - hooks rope

let go of space ("up" state) - releases rope

**Comment by BujuArena** [comment]
This doesn't modify any external program.  It uses Windows API calls to add input messages to the message queue.  It doesn't change what the initial press does, and doesn't remove messages from the message queue.
